src/vmm/src/cpu_config/x86_64/cpuid/amd/normalize.rs

```diff
@@ -107,7 +108,7 @@ impl super::AmdCpuid {
         self.update_largest_extended_fn_entry()?;
         self.update_extended_feature_fn_entry()?;
         self.update_amd_feature_entry(cpu_count)?;
-        self.update_extended_cache_topology_entry(cpu_count, cpus_per_core)?;
+        self.update_extended_cache_topology_entry(cpus_per_core)?;
```

Comment:
I only looked at the commit "Edit CPUID Normalization to account for possibility of hotplugging", so this might be a basic question, but why can't we update it when hotplugging vcpus instead of boot time?
Software running inside guests (like guest kernel) only checks it at boot time and allocates memory to hold some data that is not extensible afterwards? Or we just don't want to update CPUID/MSR of the existing vcpus?

Reply:
The changes that I've made are to the "maximum values". I.e. maximum APIC ID sharing L3 cache, and maximum APIC address per physical package. I'm not sure if this reserves memory at boot time or not. 

From my understanding, the effect of this is that any APIC address that falls within that range is assigned to the cache/package. So all that results from this change is we say that if more vCPUs are added, they will all fall under the current cache/package.  For this feature branch at least, hot-plugging is always available, so I think it makes sense for these fields to be set from boot, so that they don't have to changed later

Reply:
Fair enough. Sounds good to me :)